src/app/services/rider.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http'; 
+import { Observable } from 'rxjs';
+import { User } from '../models/user.model';
+
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RiderService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  private url: string = "http://localhost:8080/users/"; 
```

Comment:
This value should change based on different environments, we'll need to set up different environment files and reference URL's from those rather than hardcoding them here